krcal/core/kr_parevol_functions.py

```diff
@@ -147,19 +154,18 @@ def computing_kr_parameters(data       : pd.DataFrame,
     return pars
 
 
-def kr_time_evolution(ts         : np.array,
-                      masks_time : List[np.array],
-                      dst        : pd.DataFrame,
-                      emaps      : ASectorMap,
-                      xr_map     : Tuple[float, float],
-                      yr_map     : Tuple[float, float],
-                      nx_map     : int,
-                      ny_map     : int,
-                      zslices_lt : int                 = 50,
-                      zrange_lt  : Tuple[float,float]  = (0, 550),
-                      nbins_dv   : int                 = 35,
-                      zrange_dv  : Tuple[float, float] = (500, 625),
-                      detector   : str                 = 'new')->pd.DataFrame:
+def kr_time_evolution(ts            : np.array,
+                      masks_time    : List[np.array],
+                      dst           : pd.DataFrame,
+                      emaps         : ASectorMap,
+                      bootstrap_map : ASectorMap,
```

Comment:
Are we gonna use both a provided and a bootstrap?

Reply:
Some of my comments were not published, sorry. I said that I'm not sure if it matters which one we use, but let's keep it as it is to
- minimize changes wrt to the previous version
- finish the job asap